crates/sui-json-rpc/src/transaction_builder_api.rs

```diff
@@ -112,6 +112,35 @@ impl RpcTransactionBuilderServer for FullNodeTransactionBuilderApi {
         Ok(TransactionBytes::from_data(data)?)
     }
 
+    async fn pay_sui(
```

Comment:
Can we add comments about the what these endpoints do as well as all the arguments? Here's a good example:

https://github.com/MystenLabs/sui/blob/b6d56bd22a6076cd4cd1d1cca657da21a6ae2f67/crates/sui-json-rpc/src/api.rs#L208-L223

Reply:
the corresponding function headers in api.rs have some comments for pay_sui and pay_all_sui, here are implementations of those?

Reply:
sorry commented in the wrong place. Added a new comment

Reply:
I am adding some desc for the function itself.